src/splats-mesh.ts

```diff
@@ -4,83 +4,45 @@ import {
   RawShaderMaterial,
   InstancedBufferGeometry,
   BufferGeometry,
-  InstancedBufferAttribute,
-  RGBAIntegerFormat,
-  UnsignedIntType,
-  DataTexture,
-  RGBAFormat,
-  FloatType,
-  RGFormat,
   BufferAttribute,
   Matrix4,
   Vector3,
-  RedIntegerFormat,
   Camera,
   Quaternion,
-  Texture,
   ShaderMaterial,
   DoubleSide,
   GLSL3,
   Object3D,
-  NearestFilter,
-  RGIntegerFormat,
   Vector4,
 } from 'three';
 
 import { createSortWorker } from './workers/SortWorker';
 import { PointCloudMaterial } from './materials';
+import { GeometryData } from './geometryData';
 
-const DELAYED_FRAMES = 1;
-
+const DELAYED_FRAMES = 2;
 export class SplatsMesh extends Object3D {
   public mesh: any;
   public material: ShaderMaterial | null = null;
-  public forceSorting: boolean = true;
+  private forceSorting: boolean = true;
```

Comment:
This may be a breaking change. We should probably change the major version of the library when we release it.